src/client/src/shell/routes/ShellRoute.tsx

```diff
@@ -88,7 +88,7 @@ const portalProps = {
     config: {
       name: 'analytics',
       width: 400,
```

Comment:
Do you also need to increase the `width` if you increased padding?

Reply:
Yeah, I chose that over changing the property of the Analytics Container (overflow-y) because of the convention that all other components in pop out mode do not have a overflow-y set to auto but hidden.